src/quo/components/profile/collectible/view.cljs

```diff
@@ -14,12 +16,18 @@
     (str "+" amount)]])
 
 (defn tile
-  [{:keys [style resource size]}]
-  (let [source (if (string? resource) {:uri resource} resource)]
+  [{:keys [style size resource]}]
+  (let [svg? (and (map? resource) (:svg? resource))]
     [rn/view {:style style}
-     [rn/image
-      {:style  (style/tile-style-by-size size)
-       :source source}]]))
+     (if svg?
+       [svg/svg-uri
+        {:uri   (:uri resource)
+         :style (style/tile-style-by-size size)}]
+       [fast-image/fast-image
+        {:style  (style/tile-style-by-size size)
+         :source (if (string? resource)
+                   {:uri resource}
+                   resource)}])]))
```

Comment:
This component needs to be reimplemented, the collectible component has changed, in the meantime I just added support to show svgs